src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,58 @@
+.nav {
+  display: flex;
+  margin-right: 50px;
+  min-height: 60px;
```

Comment:
move this styles to `header__nav` [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 8th item